package.json

```diff
@@ -43,6 +46,7 @@
     "chai": "^4.3.7",
     "cross-env": "^7.0.3",
     "dotenv": "^16.0.1",
+    "elliptic": "^6.6.1",
```

Comment:
I believe this addition is not necessary. There was no direct dependency on `elliptic`